recipes/cythonbiogeme/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "cythonbiogeme" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cythonbiogeme-{{ version }}.tar.gz
+  sha256: f139342944c849b47ce5aaa548e68aa179c195de93ab76be78353f4309684d0e
+
+build:
+  number: 3
+  script: /bin/rm -f pyproject.toml && {{ PYTHON }} -m pip install . -vv  # [unix]
+  skip: true  # [python_impl == 'pypy']
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+  host:
+    - python
+    - setuptools >=63.4.1
+    - wheel
+    - numpy >=1.19.0
+    - cython
+    - pip
+  run:
+    - python
+    - cython >=0.29.16
```

Comment:
Shouldn't need this a run dependency, remove.

Reply:
Nevermind I see it in the setup.cfg:

install_requires =
    cython >= 0.29.16
    pandas >= 1.3.5


Does the package compile things at run time?